preprocessing.py

```diff
@@ -169,6 +169,7 @@ def get_input_tensors(batch_size, tf_records, num_repeats=None,
 
     Returns a dict of tensors (see return value of batch_parse_tf_example)
     '''
+    print ("Reading tf_records from {} inputs".format(len(tf_records)))
```

Comment:
useful sanity check when training not with chunks to make sure your holdout games have been excluded.